### PR TITLE
Edge-case fixes for is_invalid function

### PR DESCRIPTION
**Pull request by kongzii**

### CODE REVIEW

File: tests/tools/test_is_invalid.py [hunk 1]
```diff
@@ -73,6 +73,14 @@
...
             True,  # Invalid because it's about ethicalness.
         ),
+        (
+            "In the next 12 months will the GenAI scaling craze see a group spend >$1B to train a single large-scale model?",
+            True,  # Invalid because it's relative timing that depends on "now".
+        ),
+        (
+            "In the next 12 months will an AI-generated song break into the Billboard Hot 100 top 10 or the Spotify Top Hits?",
+            True,  # Invalid because it's relative timing that depends on "now".
+        ),
```
Comment (kongzii) [inline tests/tools/test_is_invalid.py:83]: All green now. These two would fail previously

Screenshot by Dropbox Capture

**Review by evangriffiths — APPROVED** [review]
